examples/lib/stories/experimental/layout_component_example_1.dart

```diff
@@ -127,36 +127,40 @@ class LayoutDemo1 extends LinearLayoutComponent {
   }) {
     return [
       TextComponent(text: 'Some short text'),
-      if (expandedMode)
-        ExpandedComponent(
-          child: RectangleComponent(
-            size: Vector2(100, 70),
-            paint: Paint()..color = Colors.amber,
-          ),
-        )
-      else
-        RectangleComponent(
+      componentWrapper(
+        builder: (child) {
+          if (!expandedMode) {
+            return child;
+          }
+          return ExpandedComponent(child: child);
+        },
+        child: RectangleComponent(
           size: Vector2(100, 70),
           paint: Paint()..color = Colors.amber,
         ),
-      if (expandedMode)
-        ExpandedComponent(
-          child: PaddingComponent(
-            padding: padding,
-            child: RectangleComponent(
-              size: Vector2(96, 96),
-              paint: Paint()..color = Colors.blue,
-            ),
-          ),
-        )
-      else
-        PaddingComponent(
+      ),
+      componentWrapper(
+        builder: (child) {
+          if (!expandedMode) {
+            return child;
+          }
+          return ExpandedComponent(child: child);
+        },
+        child: PaddingComponent(
           padding: padding,
           child: RectangleComponent(
             size: Vector2(96, 96),
             paint: Paint()..color = Colors.blue,
           ),
         ),
+      ),
     ];
   }
+
+  static Component componentWrapper({
```

Comment:
If possible, it would be good to be very explicit in these examples and not use wrappers, so that they are easier to read for beginners.

Reply:
Haha, sorry for giving you conflicting comments. :sweat_smile:

Reply:
Hello! I hate to pester you... but any chance I can get a review? :3